introduction/problems-solved.md

```diff
@@ -0,0 +1,35 @@
+
+# What Problems Does InnerSource Solve?
+
+To illustrate the types of situations where inner source can help, imagine the following.
+Suppose two teams at the same company deliver separate pieces of software with one team's software depending on that of the other.
+An example could be a user experience that depends on an API service to retrieve data for display.
+This situation is common in a large enterprise where a single team producing software may have dozens or hundreds of consumers.
+
+When consuming teams need many features, producing teams normally have some sort of requirements and prioritization process to decide which features they will work on.
+For critical feature requests that are not prioritized for immediate work, the consuming team may commonly choose one of 3 options, each of which come with their own drawbacks.
+
+1. **Wait it out**. The consuming team may do nothing and limp along without the requested functionality.
+  This option requires the least amount of work on their side.
+  Depending on the benefit of the feature request, waiting might be just fine.
+  However, it may come with real amounts of pain, especially if the requested functionalty is never delivered.
+1. **Workaround**. A consuming team that doesn't want to wait may do extra work somewhere else to compensate for the absence of their requested feature.
+  This extra work may come as change in the consuming project.
+  Alternatively, they may create a new project that meets their needs and replaces their use of all or part of the producing team's system (code/project duplication).
+  This strategy allows the consuming team to get the requested feature via their own efforts only (no hard dependencies). It comes with several drawbacks, though.
+      
+    1. Any work done by the consuming team remains unavailable to any other consumers with the same feature request.
+    1. The consuming team has inadvertently signed up for the long-term burden of maintaining the newly-written code, which is not in the domain of their core team competency.
+    1. The company as-a-whole acquires duplicate projects and code in the same problem space.
+      
+1. **Escalate**. The consuming team may not take "no" for an answer and, instead, advocate to someone in the producers' management hierarchy to influence (or force) the producing team to do the work.
+This option sounds attractive for the consuming team because they get the requested feature without doing the work to implement or maintain it.
+It is still a drag on the team, though, because it necessarily diverts some of their attention and work to the non-engineering task of escalation.
+Additionally, this option does not scale as there are so many times that a consumer can escalate feature requests before damaging their credibility.
```

Comment:
I'm not a native speaker, of course, but shouldn't we `s/there are so many times/there are _only_ so many times`?

Reply:
Apparently you are a better speaker than I am 😜